tests/integration/modules/test_pkg.py

```diff
@@ -326,3 +326,24 @@ def test_pkg_upgrade_has_pending_upgrades(self):
                 self.assertNotEqual(ret, {})
                 if 'changes' in ret:
                     self.assertNotEqual(ret['changes'], {})
+
+    @destructiveTest
+    @skipIf(salt.utils.is_windows(), 'minion is windows')
+    def test_pkg_latest_version(self):
+        '''
+        check that pkg.latest_version returns the latest version of the uninstalled package (it does not install the package, just checking the version)
+        '''
+        grains = self.run_function('grains.items')
+        cmd_info = self.run_function('pkg.info_installed', ['htop'])
+        if cmd_info != '':
```

Comment:
I'm assuming you are checking for a `''` return if the package htop is not installed? If that assumption is correct this means this test will fail if the package is not installed as it wont get a value from cmd_htop as it doesn't get into this if statement.